seqr/views/apis/project_api.py

```diff
@@ -306,7 +306,7 @@ def _get_json_for_variant_tag_types(project):
         })
 
     project_variant_tags.append(note_tag_type)
-    project_variant_tags = sorted(project_variant_tags, key=lambda variant_tag_type: variant_tag_type['order'])
+    project_variant_tags = sorted(project_variant_tags, key=lambda variant_tag_type: 0 if variant_tag_type['order'] is None else variant_tag_type['order'])
```

Comment:
Without this change, there is an error: `TypeError: '<' not supported between instances of 'NoneType' and 'float'` in Python 3.

Reply:
1) `order` isn't ever `None` so this should be impossible to hit, which concerns me
2) If you really need to change this `variant_tag_type['order'] or 0` is neater

Reply:
There is one of the variant_tag_type's which variant_tag_type['order'] is None. We should either fix the data in the database or update the code.

Reply:
Okay, update the code to `variant_tag_type['order'] or 0`